models/issues/comment.go

```diff
@@ -181,40 +182,40 @@ func (t CommentType) HasAttachmentSupport() bool {
 	return false
 }
 
+type Role string
+
 // RoleDescriptor defines comment tag type
-type RoleDescriptor int
+type RoleDescriptor struct {
+	IsPoster util.OptionalBool
```

Comment:
```suggestion
	IsPoster bool
```

`OptionalBool` doesn't seem useful

Reply:
Done.